ffac-update-location-gps/files/etc/hotplug.d/usb/10-update-location-gps

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+TTYPATH="/sys${DEVPATH}/tty"
+LOCKPATH="/tmp/update-location-gps"
+
+ENABLED=$(uci get update-location-gps.settings.enabled)
+
+if [ "${ACTION}" = "bind" ]; then
+	[ "${ENABLED}" != "1" ] && exit 0
```

Comment:
Why are we checking first for the action and then for enabled? Shouldn't the code exit immediately if it was disabled? If not, maybe a comment explaining what the expected flow is, would be great.

Reply:
I did copy this from 10-usb-wan-hotplug - the Idea is to execute the unbind action even if the lockfile exist but the setting is disabled